src/Elmish/Hooks/UseEffect.js

```diff
@@ -0,0 +1,17 @@
+const React = require("react")
+
+class LifeCycles extends React.Component {
```

Comment:
Better give it a name that points to `useEffect` somehow. This name will pop up in React's error messages.

Reply:
But long-term, I think we would benefit from an all-purpose class like this built into Elmish itself and exposing all (or most) lifecycle methods as props.